client/test/languageServer/formatter.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as vscode from "vscode";
+
+import * as assert from "assert";
```

Comment:
I notice `chai` has an assert function as well. Why do we use this one over the chai version?

Reply:
The "assert" is a built-in module. I would not add dependency until there's a need.